addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/gui/ParamMinerDialog.java

```diff
@@ -138,6 +188,12 @@ public void save() {
         config.setUsePredefinedCookieWordlists(this.getBoolValue(PREDEFINED + COOKIE_WORDLIST));
 
         config.setUseCustomUrlWordlists(this.getBoolValue(CUSTOM + URL_WORDLIST));
+        if (config.getUseCustomUrlWordlists()) {
+            logger.info(
+                    "Custom url wordlist path: {}",
```

Comment:
Full caps on `URL` :grinning:

Reply:
Well, it should be logger.debug anyway I guess

Reply:
It should be debug, is it really necessary though?